src/main/java/com/cdancy/jenkins/rest/JenkinsClient.java

```diff
@@ -52,22 +57,26 @@ public JenkinsClient() {
      */
```

Comment:
I forgot to add and `@param` annotation, but I am not sure what to say about what modules provide other than logging.

Reply:
This is fine. The modules themselves are generic and so there is not much more one can say about them.